packages/app/src/components/DonationDialog.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <q-dialog v-model="inDonation">
+      <q-card class="donation-card">
+        <q-card-section>
+          <h6>Thank you for donating! 💕</h6>
+          <span>Your donations help keep our work alive.</span>
+          <span><br>All proceeds go to Grey Software.</span>
+        </q-card-section>
+        <q-card-section>
+          <div class="column items-center">
+            <q-btn
+              class="q-pa-xs q-ma-sm"
+              icon="img:../statics/icons/paypal-28x28.png"
```

Comment:
Please use the mdi-paypal icon

Reply:
uh... there isn't a native `mdi-paypal` icon that ships with quasar. the only mention of one i could find is here

https://themesdesign.in/xoric/layouts/blue/vertical/icons-materialdesign.html

but this isn't the mdi icon set quasar ships with.

Reply:
The website's new navbar has a paypal icon! https://github.com/grey-software/grey.software/blob/master/assets/icons/paypal.svg